src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx

```diff
@@ -60,13 +62,8 @@ export const AttributeGeneralSettings = () => {
     defaultValue: false,
   });
 
-  useFetch(
-    () => adminClient.clientScopes.find(),
-    (clientScopes) => {
-      setClientScopes(clientScopes);
-    },
-    []
-  );
+  useFetch(adminClient.clientScopes.find, setClientScopes, []);
+  useFetch(adminClient.users.getProfile, setConfig, []);
```

Comment:
Again, not a fan of making this implicit.
```suggestion
  useFetch(() => adminClient.clientScopes.find(), setClientScopes, []);
  useFetch(() => adminClient.users.getProfile(), setConfig, []);
```

Reply:
we do this everywhere though, if you feel strongly about we should agree on this and make a code style rule for it